platform_API.md

```diff
@@ -50,6 +50,19 @@ Authorization: Key <your Server API Key>
 
 ## API Reference
 
+### Authentication
+
```

Comment:
Might be good to include a "Base path: `/me`" similar to the Payments documentation below to improve consistency.

Reply:
I could be wrong but I don't think we plan to use `/me` as the base path.

Reply:
True, it seems like there wouldn't be any url parameters so it's probably fine without a base path.

Reply:
Nice
I wish i understand about this